_translations/index_se.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: signed
+title: Ett öppet brev till stöd för Richard M. Stallman
+description: Ett öppet brev till stöd för Richard Matthew Stallman återinförs av Free Software Foundation
+image: /assets/social-media-preview_se_SE.png
+locale: se_SE
+twitter:
+  card: summary_large_image
+---
+
+2021-03-23
+
+Richard M. Stallman, ofta känd som RMS, har varit en drivande
+kraft i den fria programvarurörelsen i årtionden vars bidrag inkluderar
+operativsystemet GNU och Emacs.
+
+Nyligen har det gjorts avskyvärda onlineattacker som vill undanröja
+honom från FSFs styrelse eftersom han har uttryckt sina personliga
+åsikter. Vi har sett detta hända tidigare på ett organiserat
+sätt med andra framstående gratisprogramvaruaktivister och
+programmerare. Vi kommer inte längre att vara tysta när
+en ikon för denna gemenskap attackeras.
+
+FSF är ett självständigt organ som behandlar sina medlemmar på
+ett rättvist, opartiskt sätt och bör inte ge efter för yttre
+sociala påtryckningar. Vi uppmanar FSF att objektivt överväga argumenten
+mot RMS och att verkligen förstå innebörden av hans
+ord och handlingar.
+
+Historiskt har RMS uttryckt sina åsikter på sätt som upprör
+många människor. Han är vanligtvis mer fokuserad på den
+filosofiska grunden, att sträva efter den objektiva sanningen
+och språklig purism, samtidigt som han understryker
+människors känslor i frågor som han uttalar sig om.
```

Comment:
According to machine translation: _while emphasizing
people's feelings on issues he speaks about._

This seems the opposite of _while underemphasising people’s feelings on matters he’s commenting on._